src/ng/interpolate.js

```diff
@@ -338,7 +371,7 @@ function $InterpolateProvider() {
 
               return compute(values);
             } catch (err) {
-              $exceptionHandler($interpolateMinErr.interr(text, err));
+              $exceptionHandler(err);
```

Comment:
I remember why now : this wraps noconcat errors in an interpolate:interr error, which seems sane, so I've kept the wrapping as it was before this PR.